ansible_collections/arista/avd/molecule/cv_deploy/intended/configs/base_configs/avd-ci-leaf1.cfg

```diff
@@ -0,0 +1,188 @@
+!RANCID-CONTENT-TYPE: arista
```

Comment:
These configs should be regenerated with the latest AVD code. Optimally we should have a proper AVD inventory and run AVD first to generate the configs. That will ensure that we always test against current AVD configs.

Reply:
These are managed by AVD today. I just copy pasted the switch configs by mistake. I updated them.